yaml_files/simphony_metadata.yml

```diff
@@ -436,6 +484,50 @@ CUDS_KEYS:
     definition: Periodic boundary condition (PBC)
     models: [CUBA.ELECTRONIC, CUBA.ATOMISTIC, CUBA.MESOSCOPIC, CUBA.CONTINUUM]
 
+  SHEAR_STRESS_POWER_LAW_SLIP_VELOCITY:
+    parent: CUBA.CONDITION
+    definition: Shear stress power law dependant slip velocity boundary condition
+    models: [CUBA.CONTINUUM]
+    CUBA.VARIABLE:
```

Comment:
It's an nonlinear boundary condition for wall tangential velocity of the form
v_s = k S^n
where k and n are parameters and v_s is the slip velocity (tangential velocity on the wall) and S is the wall shear stress